src/content/developers/docs/transactions/index.md

```diff
@@ -174,11 +174,10 @@ Once the transaction has been submitted the following happens:
 1. Once you send a transaction, cryptography generates a transaction hash:
    `0x97d99bc7729211111a21b12c933c949d4f31684f1d6954ff477d0477538ff017`
 2. The transaction is then broadcast to the network and included in a pool with lots of other transactions.
-3. A miner must pick your transaction and include it in a block in order to verify the transaction and consider it "successful".
-   - You may end up waiting at this stage if the network is busy and miners aren't able to keep up.
-4. Your transaction will receive "confirmations". The number of confirmations is the number of blocks created since the block that included your transaction. The higher the number, the greater the certainty that the network processed and recognized the transaction.
-   - Recent blocks may get re-organized, giving the impression the transaction was unsuccessful; however, the transaction may still be valid but included in a different block.
-   - The probability of a re-organization diminishes with every subsequent block mined, i.e. the greater the number of confirmations, the more immutable the transaction is.
+3. A validator must pick your transaction and include it in a block in order to verify the transaction and consider it "successful".
+4. As time passes the block containing your transaction will be upgraded to "justified" then "finalized". These upgrades make it much
+   more certain that your transaction was successful and will never be altered. Once a block is "finalized" it could only ever be changed
+   by an attack that would cost many billions of dollars.
 
 ## A visual demo {#a-visual-demo}
```

Comment:
Should we remove this video @jmcook1186?

Reply:
yea I think so
```suggestion
```

Reply:
This is non-blocking, let's chat through external resources at a different point.